src/components/Header/Header.tsx

```diff
@@ -36,12 +36,13 @@ const Header = ({
   const handleToggle = () => {
     if (navbarStatus === true) {
```

Comment:
this all could be replaced with `changeNavbarStatus(!navbarStatus)`. in general, you shouldn't leave comments/old code. if you need to readd it, you can look up the files history